doctrine/doctrine-migrations-bundle/1.2/etc/packages/migrations.yaml

```diff
@@ -0,0 +1,5 @@
+doctrine_migrations:
+    dir_name: '%kernel.project_dir%/DoctrineMigrations'
+    namespace: App\Migrations
```

Comment:
`dir_name` defines the path where migrations are looked for when loading them (and the place where they are generated). I agree with you that the name is unfortunate as it is a path.
`namespace` defines the namespace used when generating new migrations (you never create the boilerplate code of migrations by hand unless you are crazy, as the bundle has CLI commands for that).

And we need to define them explicitly here, because the recipe decided to use non-default values.